must-gather/collection-scripts/gather

```diff
@@ -21,6 +21,10 @@ BASE_COLLECTION_PATH="${1:-/must-gather}"
 mkdir -p "${BASE_COLLECTION_PATH}"
 cd $BASE_COLLECTION_PATH
 
+# define a local cache for kube API to improve the discovery and avoid client-side throttling
```

Comment:
Would this cause the cache to be collected in the must-gather output? If so, I'd prefer to just put the cache in /tmp somewhere.

Reply:
Do you anticipate any issues? Could the cache be useful to debug/triage?

Reply:
> Would this cause the cache to be collected in the must-gather output?

Yes, the proposed cache-dir was sitting next to the must-gather output.

Addressed in https://github.com/openshift/custom-metrics-autoscaler-operator/pull/23/commits/ffedbc7a7f7a7cf3d3f4b040bcfa186793fa05b7

Reply:
I'm not certain, but I don't think that the cache would be very useful for debugging.